docs/table/operations.rst

```diff
@@ -934,3 +937,43 @@ Using multiple columns as ``keys``::
    M31 2012-02-14  16.9  17.3
    M82 2012-02-14  16.2  14.5
    M82 2012-03-26  15.7  16.5
+
+
+.. _set-difference:
+
+Set difference
+--------------
+
+A set difference will tell you the elements that are contained in one set but
+not in the other.  This concept can be applied to rows of a table by using the
+`~astropy.table.setdiff` function. You provide the function with two input
+tables and it will return all unique rows in the first table. If no unique
+rows are found, the `~astropy.table.setdiff` function will return an empty
+table.
+
+The table meta-data is merged with the `~astropy.table.join` function. To
```

Comment:
You can drop the sentence about `table meta-data`, since in fact the meta data is not merged (it just copies the table1 metadata and ignores table2 metadata completely).